src/rv/ui/screens/ViewerScreenBase.java

```diff
@@ -34,9 +34,9 @@
 import rv.world.WorldModel;
 import rv.world.objects.Agent;
 
-public abstract class ViewerScreenBase extends ScreenBase
-        implements KeyListener, MouseListener, MouseMotionListener, MouseWheelListener,
-        GameState.GameStateChangeListener, WorldModel.SelectionChangeListener {
+public abstract class ViewerScreenBase extends ScreenBase implements KeyListener, MouseListener,
```

Comment:
There are a few formatting changes in this file that are probably caused by you using an older Eclipse version (the formatter has been rewritten in the latest version, Eclipse Mars, and behaves a bit differently - I reformatted everything in latest Eclipse here: https://github.com/magmaOffenburg/RoboViz/commit/73d7e79542bb47d2b8aedcac65e4fa3a340377a2).